dlt/common/destination/reference.py

```diff
@@ -190,6 +190,8 @@ class DestinationClientDwhConfiguration(DestinationClientConfiguration):
     """How to handle replace disposition for this destination, can be classic or staging"""
     staging_dataset_name_layout: str = "%s_staging"
     """Layout for staging dataset, where %s is replaced with dataset name. placeholder is optional"""
+    dataset_name_normalization: bool = True
```

Comment:
I think an imperative makes more sense her "normalize_dataset_name"

Reply:
yes, but we already have this field, and I don't want to change it in all places
```
def normalize_dataset_name(self, schema: Schema) -> str:
```

I was thinking of `to_normalize_dataset_name`, but we don't use such a naming in `dlt`. Do you have any ideas for naming?

Reply:
ok then `enable_dataset_name_normalizaton`?